Open_Acidification_pH-stat_arduino/Open_Acidification_pH-stat_arduino.ino

```diff
@@ -49,20 +49,35 @@ EthernetServer ethernet_server(80);
 int request_char_counter = 0;  // for parsing html requests
 String data;                   // GET query with data
 
-float interval = 1200000;  // interval at which to update Google Sheets (milliseconds)
+// Time Constants /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+int const SECOND_IN_MILLIS = 1000;
+int const MINUTE_IN_MILLIS = 60 * SECOND_IN_MILLIS;
+int const HOUR_IN_MILLIS = 60 * MINUTE_IN_MILLIS;
+int const DAY_IN_MILLIS = 24 * HOUR_IN_MILLIS;
+
+int const ONE_SECOND_DELAY_IN_MILLIS = 1 * SECOND_IN_MILLIS;
+int const THREE_SECOND_DELAY_IN_MILLIS = 3 * SECOND_IN_MILLIS;
+int const FIVE_SECOND_DELAY_IN_MILLIS = 5 * SECOND_IN_MILLIS;
+int const SEVEN_SECOND_DELAY_IN_MILLIS = 7 * SECOND_IN_MILLIS;
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+int interval = 20 * MINUTE_IN_MILLIS;  // interval at which to update Google Sheets (milliseconds)
```

Comment:
Use an `unsigned long` for holding millis time variables.

Same for `CHILLER_INTERVAL`, `sensor_interval`, and `LEASE_INTERVAL`.